tests/extras/webpack.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern!bdd',
+  'intern/chai!assert',
+  'intern/dojo/node!child_process'
+], function (bdd, assert, child_process) {
+  'use strict';
+
+  var exec = child_process.exec;
+
+  bdd.describe('webpack', function () {
+    bdd.it('works with webpack', function () {
+      var dfd = this.async(5000);
+
+      var WEBPACK_CMD = './node_modules/.bin/webpack tests/extras/fixtures/app.js .tmp/bundle.js';
+      var BUNDLE_CMD = 'node .tmp/bundle.js';
+
+      exec(WEBPACK_CMD, function (err) {
+        if (err) {
+          return dfd.reject(err);
+        }
+
+        exec(BUNDLE_CMD, function (err, stdout, stderr) {
+          if (err) {
+            return dfd.reject(err);
+          }
+          assert.equal(stdout, 'ok\n');
+          return dfd.resolve();
+        });
+      })
```

Comment:
I did a quick scan of the ESLint errors after adding the /tests/* directory and most were just single-vs-double quotes, or missing semicolons, or how it doesn't like strict mode.

But after tweaking all those locally, these were the only interesting errors:
```sh
$ grunt lint
Running "eslint:config" (eslint) task
Running "eslint:app" (eslint) task
Running "eslint:tests" (eslint) task

tests/spec/auth/api.js
   56:17  error  "p" is not defined                            no-undef
   79:10  error  configWithoutEmail is defined but never used  no-unused-vars
  119:17  error  "p" is not defined                            no-undef
  175:17  error  "p" is not defined                            no-undef
  232:17  error  "p" is not defined                            no-undef

✖ 5 problems (5 errors, 0 warnings)
```